docs/config/index.md

```diff
@@ -382,6 +382,18 @@ To get TypeScript working with the global APIs, add `vitest/globals` to the `typ
 }
 ```
 
+Sometimes the configuration mentioned above might not work because your workspace cannot find the types for `vitest/globals`. If that's the case, you can add `typeRoots` to your `tsconfig.json` file. Afterward, try restarting your TypeScript server. For example in VSCode, press `F1` and select `TypeScript: Restart TS server`
+
+```json
+// tsconfig.json
+{
+  "compilerOptions": {
+    "types": ["vitest/globals"],
+    "typeRoots": ["./node_modules/@types/", "./types", "./node_modules"]
```

Comment:
In what cases doesn't the default `typeRoots` work? Could you setup a minimal reproduction with https://vitest.new/ ?

Reply:
I think it's IDE specific. @AriPerkkio